sigstore/_cli.py

```diff
@@ -122,12 +162,28 @@ def _sign(
             ctfe_pem=ctfe_pem,
         )
 
-        click.echo("Using ephemeral certificate:")
-        click.echo(result.cert_pem)
+        sig_output: TextIO
+        if multiple_inputs and output_signature == "":
```

Comment:
Why does there need to be multiple inputs? From my reading, if I do this:
```
sigstore sign test_file.txt --output-signature --output-certificate
```
That will just write to stdout whereas I'd expect it to write to `test_file.txt.sig` and `test_file.txt.crt`

Reply:
I've pushed [6ed3395](https://github.com/sigstore/sigstore-python/pull/101/commits/6ed339521fbf9089e055cc0225858ac162829dd1) to fix this and done a quick test. Feel free to revert if I've misunderstood the intention.